common-custom-user-data-gradle-plugin/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+## [1.0] - 2021-01-11
+Initial release.
+
+[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/common-custom-user-data-gradle-plugin-1.0...HEAD
```

Comment:
I don't understand this link.

Reply:
I copied this from the Maven Extension: it doesn't work yet, but it should show changes since the last release.
In fact, it's likely the changelog will need to be updated as part of the initial release.